### DISCUSSION TITLE
Lite version systemd service not running

### DISCUSSION THREAD

**Discussion by kreptafire** [TheAssassin/AppImageLauncher#566] [Q&A]
### Pre-submit checks

- [X] I [checked for similar issues](https://github.com/TheAssassin/AppImageLauncher/issues?q=) beforehand, but could not find any, not even closed ones. I could not add my bug report to any existing issue.
- [X] I am going to take the time to to fill in all the required details. I know that the bug report will be dismissed otherwise.

### Describe the bug

Installation on the steam deck proceeds normally. appimagelauncherd.service file and symlink are in the correct place. Running sudo systemctl start appimagelauncherd fails, returning "unit not found".

### Expected behavior

I expect to see the service start.

### Steps to reproduce the issue

1. Install on the steam deck.
2. Run sudo systemctl start appimagelauncherd.

### Distribution and desktop environment

SteamOS

### Installed AppImageLauncher version

appimagelauncher-lite-2.2.0-travis995-0f91801-x86_64.appimage

**Comment by TheAssassin** [comment]
Add a `--user` to your `systemctl` command and remove `sudo`.

**Comment by kreptafire** [comment]
Thanks! That worked. How do I make sure the service starts automatically? I was thrown off because I don't seem to get prompts when I add an appimage file to the applications directory or when I execute an appimage file.

**Reply by TheAssassin** [reply]
Whenever AppImageLauncher runs, it will check whether the service is running, and start/stop it based on your configuration. By default, the service will be started.

The lite version only works on non-executable files, i.e., if an AppImage is executable, you won't get a prompt.
